homeassistant/components/sonos/__init__.py

```diff
@@ -173,8 +173,7 @@ def _create_soco(self, ip_address: str, source: SoCoCreationSource) -> SoCo | No
 
     async def _async_stop_event_listener(self, event: Event) -> None:
         await asyncio.gather(
-            *(speaker.async_unsubscribe() for speaker in self.data.discovered.values()),
-            return_exceptions=True,
+            *(speaker.async_unsubscribe() for speaker in self.data.discovered.values())
         )
```

Comment:
Should we keep the `return_exceptions=True` and log them here as well?  If more than one speaker has an exception the first one will raise and the rest of them will be uncaught

Reply:
The coroutine it calls should not be able to throw exceptions to this level as it gathers & returns them:
https://github.com/home-assistant/core/blob/1ea6955fbf8b2719332ea2907d46141a288fbbfc/homeassistant/components/sonos/speaker.py#L346-L356

Reply:
Edit: oh, I think I see what you're saying. Let me review.

Reply:
Ahh, I see you are already catching them one level deeper.